state-chain/chains/src/btc.rs

```diff
@@ -1388,4 +1385,12 @@ mod test {
 		assert!(!ConsolidationParameters::new(1, 1).are_valid());
 		assert!(!ConsolidationParameters::new(0, 10).are_valid());
 	}
+
+	#[test]
+	#[should_panic]
+	// XXX: @martin-chainflip 👋
```

Comment:
@martin-chainflip 👋

Reply:
Also we should also be using saturating_add really when adding here: `self.btc_fee_info.min_fee_required_per_tx + self.btc_fee_info.fee_per_output_utx`